### DISCUSSION TITLE
puttyNG 0.78 size font is too big with same configuration as 0.71

### DISCUSSION THREAD

**Discussion by shinji7800** [mRemoteNG/mRemoteNG#2457] [Q&A]
hello everyone 

I updated PuttyNG beaucoup I had issue with ssh connection? 

but font size i way to big for me, do you have an idea why ? 

![puttyNG_0-71](https://github.com/mRemoteNG/mRemoteNG/assets/9131657/479ec6fd-4905-4b4c-b07f-d682544bfd4b)
![puttyNG_0-78](https://github.com/mRemoteNG/mRemoteNG/assets/9131657/63fea3d7-cd59-4b5d-aabc-dd6325a13188)

thanks for your help 

best regards

**Comment by Nickmcgit** [comment]
Under Options > Advanced > Putty options you should be able to look at the defaults for PuttyNG and see if the font is size 10 or something else. Size 10 "Console" is the default.
